### DISCUSSION TITLE
Try to contribute

### DISCUSSION THREAD

**Discussion by Youssef11296** [primer/react#3757] [General] [answered]
Hello, I wanna contribute to the react repo, but, I faced an issue on setup the code locally.
Basically, I followed the steps in the contribution, by cloning, "npm i", and when I ran it using the "npm run start" it gave a bunch of errors.
can anyone help me set up the code locally?
![react-setup-err](https://github.com/primer/react/assets/66457391/74e87e22-811d-43c2-987c-6d58924ac926)

**Comment by lesliecdubs** [comment, marked as answer]
Hi @Youssef11296  👋🏻 Thanks for your interest in contributing! 

Please take a look at [this section in our contributor docs](https://github.com/primer/react/blob/main/contributor-docs/CONTRIBUTING.md#developing-components) and make sure you have Node.js v16 installed before running the setup commands.

To check your Node.js version:
> node -v

If you don't have v16 installed:
> nvm install 16

Once Node is ready, setup the project:
> npm run setup

Let us know what happens when you give this a try.

**Comment by Youssef11296** [comment]
@lesliecdubs, thanks, for the response.
I actually did, but it shows an issue with the "./script/setup" as the "." is not a recognized command, it's related to my system.
Trying to solve it from my side.
![errr](https://github.com/primer/react/assets/66457391/c293e60d-5bff-4856-b9a7-9d70e229dd9e)

**Comment by Youssef11296** [comment]
@lesliecdubs, if you can share a help with that, it'll be totally appreciated.

**Reply by lesliecdubs** [reply]
Hm. I can't tell from the screenshot you posted above, but are you definitely running the `npm run setup` command in a local clone of the `primer/react` repo? 

I'm not familiar with this particular error and am not a regular Windows user, but I wonder if [this article I found](https://www.thetechedvocate.org/ways-to-fix-the-not-recognized-as-an-internal-or-external-command-error-in-windows/) after Googling the error you received might be helpful?
